styleguide/source/assets/scss/02-molecules/_upcoming-event.scss

```diff
@@ -0,0 +1,90 @@
+.ama__upcoming-event {
+  @include gutter($padding-bottom-half...);
+  @include gutter($padding-top-full...);
+  @include gutter($padding-left-half...);
+  border-top: 1px solid $gray-7;
+
+  @include breakpoint($bp-small) {
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  @include breakpoint($bp-med) {
+    display: flex;
+  }
+
+  &__title {
+    svg {
+      display: none;
+
+      @include breakpoint($bp-small) {
+        @include gutter($margin-right-full...);
+        display: inline-block;
+      }
+    }
+
+    @include breakpoint($bp-med) {
+      flex-grow: 1;
+    }
+  }
+
+  &__text {
+    color: $gray-50;
+    display: flex;
+
+    @include breakpoint($bp-small) {
+      margin-left: 40px;
+    }
+
+    @include breakpoint($bp-med) {
+      margin-left: 0;
+      width: 40%;
+    }
+
+    p {
+      @include gutter($margin-right-full...);
+      @include font-size($small-font-sizes);
+
+      @include breakpoint($bp-small) {
+        width: 50%;
+      }
+
+      @include breakpoint($bp-med) {
+        @include font-size($p-font-sizes);
+        line-height: 1.222em;
+      }
+    }
+  }
+}
+
+.ama__upcoming-events {
+  position: relative;
+  
+  &__header {
+    display: flex;
+  }
+
+  .ama__section-title {
+    @include breakpoint($bp-med) {
+      flex-grow: 1;
+    }
+  }
+
+  &__view-all-link {
+    @include gutter($margin-right-full...);
+    @include font-size($small-font-sizes);
+    float: right;
+
+    @include breakpoint($bp-small) {
+      position: absolute;
+      top: 10px;
+      right: 20px;
+    }
+
+    @include breakpoint($bp-med) {
+      font-size: .778em;
```

Comment:
These sizes seem arbitrary. Can we extend a header here?

Reply:
These are the sizes we were given in the requirements.